fern/getting-started/quickstart.mdx

```diff
@@ -116,7 +116,7 @@ account.
 Head over to the [GitHub App](https://app.composio.dev/app/github).
 
 1. Select the option to "Setup GitHub Integration"
-2. Click on "Create Integration"
+2. Click on "Create Integration
```

Comment:
```suggestion
2. Click on "Create Integration"
```

The closing quotation mark is missing after "Integration". This is a simple typographical error that affects readability and could confuse users following the instructions.